Lib/fontTools/varLib/__init__.py

```diff
@@ -587,8 +587,14 @@ def normalize(name, value):
 			space = {}
 			for condition in conditions:
 				axis_name = condition["name"]
-				minimum = normalize(axis_name, condition["minimum"])
-				maximum = normalize(axis_name, condition["maximum"])
+				if condition["minimum"]:
```

Comment:
wait... I think you meant `if "minimum" in condition` here, no?

Reply:
actually I don't remember, is the key always there but maybe None?

Reply:
if you're checking for None, it's better to do that explicitly because as you know, in Python `0` or (`0.0`) mean False

Reply:
ok, I checked and designspaceLib `_readConditionElements` method sets those to `None` when not specified, so you should check `if condition["minimum"] is not None`